docs/concepts/consistency.mdx

```diff
@@ -9,7 +9,7 @@ TopK supports three different consistency levels, allowing you to choose the rig
 TopK follows a write-ahead log (WAL) and compaction-based architecture:
```

Comment:
imo consistency shouldnt be in "Guides" but in some other more highlevel section along with architecture

Reply:
(cant comment below) in `Indexed Consistency` we can `It is best suited for analytical workloads and batch processing where absolute real-time consistency isn’t required.`. not sure that's right check with @marekgalovic

Reply:
I plan to add "Architecture" section in the next iteration along with more "under-the-hood" content. Taking it out into a custom standalone section now would feel a bit off. I am not against renaming the existing section to "Concepts" again but I feel like "Guides" is more common in docs world.